doc/ref_model/chapters/chapter03.md

```diff
@@ -265,3 +277,15 @@ The following diagram presents an example of the realization of the reference mo
 <p align="center"><b>Figure 3-6:</b> Reference model realization example</p>
 
 The terms Container Infrastructure Service Instance and Container Infrastructure Service Manager should be understood as defined in ETSI GR NFV-IFA 029 V3.3.1 [4]. More detailed deployment examples can be found in [Section 4.3](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#43-networking) of this Reference Model document.
+
+<a name="3.8"></a>
+## 3.8 Hardware Acceleration Abstraction
+
+<a name="3.8.1"></a>
+### 3.8.1 Types of accelerators
+
+<a name="3.8.2"></a>
+### 3.8.2 Levels of accelerator consumption
+
+<a name="3.8.3"></a>
+### 3.8.3 Decoupling applications from infrastructure with Hardware Acceleration
```

Comment:
```suggestion
### 3.8.3 Off-loading applications from infrastructure with Hardware Acceleration
```

Reply:
Thanks @pgoyal01 . It is meant as "Decoupling" per [Cloud Platform Abstraction Related Terminology](https://github.com/cntt-n/CNTT/blob/master/doc/common/glossary.md#1.6), with levels of decoupling described in text similar to that was before in ex Appendix A.